NuGet.cmake

```diff
@@ -26,15 +26,12 @@ include_guard()
 include("${CMAKE_CURRENT_LIST_DIR}/ToolchainCommon.cmake")
 
 find_program(NUGET_PATH
-    nuget nuget.exe
-    PATHS
-        ${NUGET_PATH}
+    NAMES nuget nuget.exe
 )
 
 # If NuGet isn't found, download it.
 #
-# NuGet.exe will be downloaded to `TOOLCHAIN_TOOLS_PATH`. If `TOOLCHAIN_TOOLS_PATH` is not set then it will
-# be downloaded to '${CMAKE_BINARY_DIR}/__tools'.
+# NuGet.exe will be downloaded to `TOOLCHAIN_TOOLS_PATH`. If `TOOLCHAIN_TOOLS_PATH` is not set then it will be downloaded to '${CMAKE_BINARY_DIR}/__tools'.
```

Comment:
different meaning of `TOOLCHAIN_TOOLS_PATH` here?

Reply:
Tracked as #50.